docs/testdoc2.md

```diff
@@ -0,0 +1,2 @@
+# File created by contributor and then issues a pull request
+This is file2
```

Comment:
Comment from maintainer as part of the pull request view. In the list of changed files, clicked inside this testdoc2.md file on a specific line.  

Here you can write line/file specific feedback for the contributor, like: "add a version history here"

Then clicked on the "Start a review" button

Reply:
contributor: Fake action: I didn't update the file, just added this comment and will then hit the "Resolve conversation" button in the Conversation thread of the pull request. Let's see what happens....